### DISCUSSION TITLE
rAudio Working on Raspberry Pi 5

### DISCUSSION THREAD

**Discussion by feebb** [rern/rAudio#1536] [General]
I tried rAudio on a Raspberry Pi 5 and it seems to work. I haven't done extensive testing to see if all the features are o.k., but I was able to set up a share from a Synology NAS and play music through a USB DAC.  WiFi worked as well.  

I put the 20231128 image on an 8gb Sandisk microSD using the latest Raspberry Pi imager. I chose Raspberry Pi 5 under Choose Device and Custom OS under Choose OS. 

Once up and running, I updated to the 20231216 release. After an hour, with no enclosure and without a heat sink or fan, the temp was about 75-76 degrees, and the CPU loads were in the 4.00 range.

Happy Holidays!
Rod

**Comment by sadays-ymm** [comment]
thanks i am trying

**Comment by Vseason** [comment]
75–76°C? Because that seems rather high, although I don’t yet have a Pi 5 with which to compare. Are you streaming, or exclusively playing high-res media?

(For reference, a Pi 3B v1.2 without heat sinks or fan typically runs in the neighborhood of 50°C after an hour, if playing local media at 24-bit 44.1 kHz.)

**Comment by sadays-ymm** [comment]
16 bit 44.1 khz flac playing is 62°C on quatmo lms-raudio
